include/manifold/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+set(MANIFOLD_PUBLIC_HDRS
+	common.h
+	iters.h
+	manifold.h
+	optional_assert.h
+	parallel.h
+	polygon.h
+	tri_dist.h
+	vec_view.h
+	)
+
+set(MANIFOLD_C_BINDING_HDRS
+	conv.h
+	manifoldc.h
+	types.h
+	)
+list(APPEND MANIFOLD_PUBLIC_HDRS ${MANIFOLD_C_BINDING_HDRS})
+
+if (MANIFOLD_CROSS_SECTION)
+	list(APPEND MANIFOLD_PUBLIC_HDRS cross_section.h)
+endif()
+
+if (MANIFOLD_EXPORT)
+	list(APPEND MANIFOLD_PUBLIC_HDRS meshIO.h)
```

Comment:
Okay, so this is what we do instead of conditionally `addsubdirectory` - yeah that does seem cleaner. I kind of liked having the src files organized into a few directories, but this should be fine too.

Reply:
If you want we can do subdirectories within src/ and just have the src/CMakeLists.txt file reference them there.

Reply:
Yeah sounds good. I think at the very least I'd like to separate `meshIO` since I really don't consider that to be a core part of Manifold. How about anything that's an optional component gets its own sub-directory?

Reply:
OK, I put meshIO and cross_section files in subdirectories.

(FWIW - meshIO may not be a core part of Manifold, but since I can't properly prepare example failure cases without it it's fairly critical to me :-P)

Reply:
Actually, that's a thought... what about maybe teaching core manifold to import and export just minimalist .glb files, since that's what's needed for debugging/testing, and have the assimp translation live as a test or sample code?

Reply:
In fact we have that, but it's in TS (gltf-io.ts) and it too depends on an external library. It's tricky because Manifold is designed to handle multiple materials, vertex properties, etc, so minimal I/O doesn't necessarily let us test everything either. Are you willing to help us maintain it?

Reply:
Actually, I might be able to do what I need on my side, without it needing to go in manifold at all - that's probably the way to go.